### PR TITLE
Testing Deploy Scripts

### LINKED ISSUE #559: ensure the deploy script is working
after checkout the dev branch if you follow the readme to deploy locally you will face several errors the goal is to fix them

### PR DESCRIPTION
**Pull request by 0x4007**
Related https://github.com/ubiquity/ubiquity-dollar/pull/417

Includes @hashedMae's latest changes. Then I rebased to development HEAD

### CODE REVIEW

File: packages/contracts/scripts/shared/helpers/loadEnv.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const loadEnv = (path: string): Env => {
   dotenv.config({ path });
   const rpcUrl = process.env.RPC_URL || "http://localhost:8545";
```
Comment (0x4007) [inline packages/contracts/scripts/shared/helpers/loadEnv.ts]: I think maybe this should be selected from our RPC selector script.

Comment (Steveantor) [reply packages/contracts/scripts/shared/helpers/loadEnv.ts]: like this @pavlovcik 
image

Comment (0x4007) [reply packages/contracts/scripts/shared/helpers/loadEnv.ts]: You should have just posted it as a suggestion on the comment lol but yes like that. I'll manually copy

**Comment by pavlovcik** [comment]
I accidentally pushed my last commit [91e5144](https://github.com/ubiquity/ubiquity-dollar/pull/554/commits/91e5144812b2b42c703db3732bbb298e48da3d71) while in a rush to leave my house. Its not stable, might need to checkout the previous commit if anybody is working off of this. Sorry

**Comment by Steveantor** [comment]
CI is now passing!!

**Pull request merged**